script.js

```diff
@@ -1,12 +1,142 @@
 //You can edit ALL of the code here
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
+  let allShow = getAllShows();
+  //Sorts shows in alphabetical order
+  allShow.sort(function (a, b) {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
```

Comment:
That's really nice function to sort out the string.  There is also a method to sort out the string. Do you come across that method?